src/WinRT.Runtime/Projections/Nullable.cs

```diff
@@ -2418,6 +2446,53 @@ unsafe object IWinRTNullableTypeDetails.GetNullableValue(IInspectable inspectabl
                 Marshal.Release(nullablePtr);
             }
         }
+
+        // Delegates are handled separately.
+        Type IWinRTNullableTypeDetails.GetNullableType() => throw new NotImplementedException();
+    }
+
+    public sealed class EnumTypeDetails<T> : IWinRTExposedTypeDetails, IWinRTNullableTypeDetails where T : struct, Enum
+    {
+        [SkipLocalsInit]

```

Comment:
Didn't we enable this for the entire module of WinRT.Runtime? Should we?

Reply:
I don't believe so, it looks to be just applied to a couple places.  We can look into whether we should enable this widely after.